04-security.md

```diff
@@ -55,17 +55,17 @@ Switch to the "Connections" tab for the Application:
 
 - **Untick the "Google" option** under "Social", as we'll just use defined username/passwords for this demo.
 
-Disable sign ups, so that only accounts that you define explicitly can sign in. Navigate to the "Authentication - Database" option in the menu, and click on the item to configure it.
+Disable sign ups, so that only accounts that you define explicitly can sign in. Navigate to the "Connections - Database" option in the menu, and click on the item to configure it.
```

Comment:
I've been using Auth0 with its "new" layout - are you currently using the original layout to get these different values?

Reply:
well look at that - there is a 'New Navigation' option - yes I was using the default. wonder how long until 'New' becomes the default

Reply:
Reverted the navigation links based on the new layout